rfcs/0002-principles.md

```diff
@@ -0,0 +1,61 @@
+---
+created-on: 2021-04-17
+freshness: 2021-04-17
+owner: @dj2
+status: proposal
+---
+
+[La version française suit.](#résumé)
+
+# Principles
+
+## Summary
+This RFC lays out the guiding principles for the developer community at CDS.
+These principles guide how we work, what we work on and why we choose projects.
+
+
+## Motivation
+In order to better work with each other and the rest of CDS it's good to lay out
+the guiding principles of the community. These principles serve as guide posts
+for the current community but also serve as a foundation for new CDS employees
+to understand how we work.
+
+
+## Proposal
+1. Optimize for outcomes
+  - Make decisions and build things that help the people served by the GC.
+  - Focus actions on working and exploring the things that matter most.
+  - Constantly be triaging and prioritizing work
+2. Do just enough, right now
+  - Do the minimum of what’s needed to test ideas
+  - Create momentum by shipping often
+  - Be agile
+  - Always work towards the Minimal Viable Product
+3. Bring people with you
```

Comment:
Work in the open to help clear a path,  Take care of each other and Put people at the heart of services

Reply:
done